Upstream: submitted
Source: https://github.com/OpenPrinting/cups-filters/pull/49

From 4b45fdd0f48ffeea211efee21d66084af12b92bc Mon Sep 17 00:00:00 2001
From: Kylie McClain <kylie@somas.is>
Date: Mon, 11 Jun 2018 18:48:22 -0400
Subject: [PATCH] Revert "driverless, foomatic-rip: Create relative symbolic
 links."

This reverts commit 0297a70022e366c56e3c8d4b1036df80874d8bb4.

The -r/--relative flag in ln is a GNU extension, and this commit adds a
somewhat unnecessary dependency on GNU coreutils. Reverting this allows for
building with other coreutils implementations, like Busybox.

---
 Makefile.am | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 6d2e32a4..cf105a7e 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1066,11 +1066,11 @@ install-exec-hook:
 	$(INSTALL) -d -m 755 $(DESTDIR)$(pkgfilterdir)
 	$(INSTALL) -d -m 755 $(DESTDIR)$(pkgbackenddir)
 if ENABLE_FOOMATIC
-	$(LN_S) -r -f $(DESTDIR)$(pkgfilterdir)/foomatic-rip $(DESTDIR)$(bindir)
+	$(LN_S) -f $(pkgfilterdir)/foomatic-rip $(DESTDIR)$(bindir)
 endif
 if ENABLE_DRIVERLESS
-	$(LN_S) -r -f $(DESTDIR)$(pkgppdgendir)/driverless $(DESTDIR)$(bindir)
-	$(LN_S) -r -f $(DESTDIR)$(pkgppdgendir)/driverless $(DESTDIR)$(pkgbackenddir)
+	$(LN_S) -f $(pkgppdgendir)/driverless $(DESTDIR)$(bindir)
+	$(LN_S) -f $(pkgppdgendir)/driverless $(DESTDIR)$(pkgbackenddir)
 endif
 if ENABLE_BRAILLE
 	$(LN_S) -f imagetobrf $(DESTDIR)$(pkgfilterdir)/imagetoubrl
